### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by pedroll**

### CODE REVIEW

**Comment by AdrianLxM** [comment]
Hi, thanks! Apparently a change in another file also has come through.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -698,24 +712,232 @@
...
-    <string name="treatments_wizard_tt_label">TT</string>
-    <string name="unsupportednsversion">Versión no acceptada de Nightscout</string>
-    <string name="enableloop">Activar loop</string>
-    <string name="disableloop">Desactivar loop</string>
-    <string name="sms_basal">Base:</string>
-    <string name="tempbasaldeliveryerror">Base temporal no emitida</string>
-    <string name="tempbasal">Base temporal</string>
-    <string name="sms_delta">Delta:</string>
-    <string name="sms_iob">IOB:</string>
+    <string name="combo_error_multiple_boluses_with_identical_timestamp">"Actual multiplicador Basal de seguridad "</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-es/strings.xml]: This seems to be a misplaced translation?
Two boluses with the same timestamp <vs.> Basal multipier?

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -698,24 +712,232 @@
...
+    <string name="readstatusfailed">Error al leer estado</string>
+    <string name="record_pump_site_change">Anotar cambio sitio bomba</string>
+    <string name="record_insulin_cartridge_change">Anotar cambio cartucho de insulina</string>
+    <string name="smbalwaysdisabled">"SMB siempre y después de que los carbohidratos  deshabilitados porque la fuente activa de BG no admite el filtrado avanzado "</string>
+    <string name="smbnotallowedinopenloopmode">"SMB no permitido en modo de Loop abierto "</string>
+    <string name="food_short">Comida</string>
+    <string name="reset">reset</string>
+    <string name="waitingfortimesynchronization">Esperando para sincronización de tiempo  (%d sec)</string>
+    <string name="loopdisconnectedfor">Desconectado (%d m)</string>
+    <string name="automatic_careportal_events">Eventos Careportal automaticos</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-es/strings.xml]: automaticos o automáticos ?

**Comment by AdrianLxM** [comment]
Thanks @pedroll. I've looked ove the translations and I really like them!
I changed back two things and added a few comments where I think there is a small typo.
One translation seems to be copied from somewhere else as it seems unrelated to the english text.
Could you please look at them?

Thanks again!

**Comment by AdrianLxM** [comment]
Thanks a lot!

**Comment by pedroll** [comment]
Hi and thanks. Sorry for the misplaced label. I have checked your observations.

**Comment by pedroll** [comment]
And sorry if pull some mistake. first time working with remotes and PR

**Pull request merged**